pkg/systemd/loginMessage.jsx

```diff
@@ -0,0 +1,50 @@
+import cockpit from "cockpit";
```

Comment:
As this is a new file, it probably requires a copyright notice

Reply:
yes, just copy it from other files, just adjust year to 2021